### ISSUE TITLE
2025-02-28, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#97]
# Today's Arxiv Papers

(1) [R2-T2: Re-Routing in Test-Time for Multimodal Mixture-of-Experts](http://arxiv.org/abs/2502.20395v1), 2025-02-27

(2) [Why Are Web AI Agents More Vulnerable Than Standalone LLMs? A Security Analysis](http://arxiv.org/abs/2502.20383v1), 2025-02-27

(3) [Multi-Agent Verification: Scaling Test-Time Compute with Multiple Verifiers](http://arxiv.org/abs/2502.20379v1), 2025-02-27

(4) [PhantomWiki: On-Demand Datasets for Reasoning and Retrieval Evaluation](http://arxiv.org/abs/2502.20377v1), 2025-02-27

(5) [Bridging Legal Knowledge and AI: Retrieval-Augmented Generation with Vector Stores, Knowledge Graphs, and Hierarchical Non-negative Matrix Factorization](http://arxiv.org/abs/2502.20364v1), 2025-02-27

(6) [Bridging the Creativity Understanding Gap: Small-Scale Human Alignment Enables Expert-Level Humor Ranking in LLMs](http://arxiv.org/abs/2502.20356v1), 2025-02-27

(7) [KEDRec-LM: A Knowledge-distilled Explainable Drug Recommendation Large Language Model](http://arxiv.org/abs/2502.20350v1), 2025-02-27

(8) [Sparse Auto-Encoder Interprets Linguistic Features in Large Language Models](http://arxiv.org/abs/2502.20344v1), 2025-02-27

(9) [Thinking Slow, Fast: Scaling Inference Compute with Distilled Reasoners](http://arxiv.org/abs/2502.20339v1), 2025-02-27

(10) [Expertise Is What We Want](http://arxiv.org/abs/2502.20335v1), 2025-02-27

(11) [Emergent Symbolic Mechanisms Support Abstract Reasoning in Large Language Models](http://arxiv.org/abs/2502.20332v1), 2025-02-27

(12) [Long-Context Inference with Retrieval-Augmented Speculative Decoding](http://arxiv.org/abs/2502.20330v1), 2025-02-27

(13) [EAIRA: Establishing a Methodology for Evaluating AI Models as Scientific Research Assistants](http://arxiv.org/abs/2502.20309v1), 2025-02-27

(14) [M^3Builder: A Multi-Agent System for Automated Machine Learning in Medical Imaging](http://arxiv.org/abs/2502.20301v1), 2025-02-27

(15) [An exploration of features to improve the generalisability of fake news detection models](http://dx.doi.org/10.1016/j.eswa.2025.126949), 2025-02-27

(16) [Judge a Book by its Cover: Investigating Multi-Modal LLMs for Multi-Page Handwritten Document Transcription](http://arxiv.org/abs/2502.20295v1), 2025-02-27

(17) [Conformal Tail Risk Control for Large Language Model Alignment](http://arxiv.org/abs/2502.20285v1), 2025-02-27

(18) [Evaluating Human Trust in LLM-Based Planners: A Preliminary Study](http://arxiv.org/abs/2502.20284v1), 2025-02-27

(19) [Large Language Models as Attribution Regularizers for Efficient Model Training](http://arxiv.org/abs/2502.20268v1), 2025-02-27

(20) [LLM as a Broken Telephone: Iterative Generation Distorts Information](http://arxiv.org/abs/2502.20258v1), 2025-02-27

(21) [Beyond Natural Language Perplexity: Detecting Dead Code Poisoning in Code Generation Datasets](http://arxiv.org/abs/2502.20246v1), 2025-02-27

(22) [FINEREASON: Evaluating and Improving LLMs' Deliberate Reasoning through Reflective Puzzle Solving](http://arxiv.org/abs/2502.20238v1), 2025-02-27

(23) [AI Will Always Love You: Studying Implicit Biases in Romantic AI Companions](http://arxiv.org/abs/2502.20231v1), 2025-02-27

(24) [ChineseEcomQA: A Scalable E-commerce Concept Evaluation Benchmark for Large Language Models](http://arxiv.org/abs/2502.20196v1), 2025-02-27

(25) [Layer-Aware Task Arithmetic: Disentangling Task-Specific and Instruction-Following Knowledge](http://arxiv.org/abs/2502.20186v1), 2025-02-27

(26) [An Extensive Evaluation of PDDL Capabilities in off-the-shelf LLMs](http://arxiv.org/abs/2502.20175v1), 2025-02-27

(27) [Re-evaluating Open-ended Evaluation of Large Language Models](http://arxiv.org/abs/2502.20170v1), 2025-02-27

(28) [Similarity-Distance-Magnitude Universal Verification](http://arxiv.org/abs/2502.20167v1), 2025-02-27

(29) [Telephone Surveys Meet Conversational AI: Evaluating a LLM-Based Telephone Survey System at Scale](http://arxiv.org/abs/2502.20140v1), 2025-02-27

(30) [Finite State Automata Inside Transformers with Chain-of-Thought: A Mechanistic Study on State Tracking](http://arxiv.org/abs/2502.20129v1), 2025-02-27

(31) [SoRFT: Issue Resolving with Subtask-oriented Reinforced Fine-Tuning](http://arxiv.org/abs/2502.20127v1), 2025-02-27

(32) [Self-Training Elicits Concise Reasoning in Large Language Models](http://arxiv.org/abs/2502.20122v1), 2025-02-27

(33) [LongRoPE2: Near-Lossless LLM Context Window Scaling](http://arxiv.org/abs/2502.20082v1), 2025-02-27

(34) [Collab-Overcooked: Benchmarking and Evaluating Large Language Models as Collaborative Agents](http://arxiv.org/abs/2502.20073v1), 2025-02-27

(35) [Polish-ASTE: Aspect-Sentiment Triplet Extraction Datasets for Polish](http://arxiv.org/abs/2502.20046v1), 2025-02-27

(36) [3D-AffordanceLLM: Harnessing Large Language Models for Open-Vocabulary Affordance Detection in 3D Worlds](http://arxiv.org/abs/2502.20041v1), 2025-02-27

(37) [AsymLoRA: Harmonizing Data Conflicts and Commonalities in MLLMs](http://arxiv.org/abs/2502.20035v1), 2025-02-27

(38) [Erasing Without Remembering: Safeguarding Knowledge Forgetting in Large Language Models](http://arxiv.org/abs/2502.19982v1), 2025-02-27

(39) [The Lookahead Limitation: Why Multi-Operand Addition is Hard for LLMs](http://arxiv.org/abs/2502.19981v1), 2025-02-27

(40) [Can Textual Gradient Work in Federated Learning?](http://arxiv.org/abs/2502.19980v1), 2025-02-27

(41) [Can Large Language Models Unveil the Mysteries? An Exploration of Their Ability to Unlock Information in Complex Scenarios](http://arxiv.org/abs/2502.19973v1), 2025-02-27

(42) [Deterministic or probabilistic? The psychology of LLMs as random number generators](http://arxiv.org/abs/2502.19965v1), 2025-02-27

(43) [SeisMoLLM: Advancing Seismic Monitoring via Cross-modal Transfer with Pre-trained Large Language Model](http://arxiv.org/abs/2502.19960v1), 2025-02-27

(44) [Collaborative Stance Detection via Small-Large Language Model Consistency Verification](http://arxiv.org/abs/2502.19954v1), 2025-02-27

(45) [GeoEdit: Geometric Knowledge Editing for Large Language Models](http://arxiv.org/abs/2502.19953v1), 2025-02-27

(46) [Playing Pokémon Red via Deep Reinforcement Learning](http://arxiv.org/abs/2502.19920v1), 2025-02-27

(47) [Meta-Reasoner: Dynamic Guidance for Optimized Inference-time Reasoning in Large Language Models](http://arxiv.org/abs/2502.19918v1), 2025-02-27

(48) [Picking the Cream of the Crop: Visual-Centric Data Selection with Collaborative Agents](http://arxiv.org/abs/2502.19917v1), 2025-02-27

(49) [LLM-driven Effective Knowledge Tracing by Integrating Dual-channel Difficulty](http://arxiv.org/abs/2502.19915v1), 2025-02-27

(50) [SkipPipe: Partial and Reordered Pipelining Framework for Training LLMs in Heterogeneous Networks](http://arxiv.org/abs/2502.19913v1), 2025-02-27

(51) [Order Doesn't Matter, But Reasoning Does: Training LLMs with Order-Centric Augmentation](http://arxiv.org/abs/2502.19907v1), 2025-02-27

(52) [Optimus-2: Multimodal Minecraft Agent with Goal-Observation-Action Conditioned Policy](http://arxiv.org/abs/2502.19902v1), 2025-02-27

(53) [Beyond the Tip of Efficiency: Uncovering the Submerged Threats of Jailbreak Attacks in Small Language Models](http://arxiv.org/abs/2502.19883v1), 2025-02-27

(54) [Towards Multimodal Large-Language Models for Parent-Child Interaction: A Focus on Joint Attention](http://arxiv.org/abs/2502.19877v1), 2025-02-27

(55) [MMKE-Bench: A Multimodal Editing Benchmark for Diverse Visual Knowledge](http://arxiv.org/abs/2502.19870v1), 2025-02-27

(56) [MIND: Towards Immersive Psychological Healing with Multi-agent Inner Dialogue](http://arxiv.org/abs/2502.19860v1), 2025-02-27

(57) [ConvCodeWorld: Benchmarking Conversational Code Generation in Reproducible Feedback Environments](http://arxiv.org/abs/2502.19852v1), 2025-02-27

(58) [ProAPO: Progressively Automatic Prompt Optimization for Visual Classification](http://arxiv.org/abs/2502.19844v1), 2025-02-27

(59) [Foot-In-The-Door: A Multi-turn Jailbreak for LLMs](http://arxiv.org/abs/2502.19820v1), 2025-02-27

(60) [Comet: Fine-grained Computation-communication Overlapping for Mixture-of-Experts](http://arxiv.org/abs/2502.19811v1), 2025-02-27

(61) [Implicit Search via Discrete Diffusion: A Study on Chess](http://arxiv.org/abs/2502.19805v1), 2025-02-27

(62) [Developmental Support Approach to AI's Autonomous Growth: Toward the Realization of a Mutually Beneficial Stage Through Experiential Learning](http://arxiv.org/abs/2502.19798v1), 2025-02-27

(63) [ChatMol: A Versatile Molecule Designer Based on the Numerically Enhanced Large Language Model](http://arxiv.org/abs/2502.19794v1), 2025-02-27

(64) [PolyPrompt: Automating Knowledge Extraction from Multilingual Language Models with Dynamic Prompt Generation](http://arxiv.org/abs/2502.19756v1), 2025-02-27

(65) [Beneath the Surface: How Large Language Models Reflect Hidden Bias](http://arxiv.org/abs/2502.19749v1), 2025-02-27

(66) [HaLoRA: Hardware-aware Low-Rank Adaptation for Large Language Models Based on Hybrid Compute-in-Memory Architecture](http://arxiv.org/abs/2502.19747v1), 2025-02-27

(67) [XCOMPS: A Multilingual Benchmark of Conceptual Minimal Pairs](http://arxiv.org/abs/2502.19737v1), 2025-02-27

(68) [R1-T1: Fully Incentivizing Translation Capability in LLMs via Reasoning Learning](http://arxiv.org/abs/2502.19735v1), 2025-02-27

(69) [Preference Learning Unlocks LLMs' Psycho-Counseling Skills](http://arxiv.org/abs/2502.19731v1), 2025-02-27

(70) [Do Expressions Change Decisions? Exploring the Impact of AI's Explanation Tone on Decision-Making](http://arxiv.org/abs/2502.19730v1), 2025-02-27

(71) [Tokens for Learning, Tokens for Unlearning: Mitigating Membership Inference Attacks in Large Language Models via Dual-Purpose Training](http://arxiv.org/abs/2502.19726v1), 2025-02-27

(72) [Few-Shot Multilingual Open-Domain QA from 5 Examples](http://arxiv.org/abs/2502.19722v1), 2025-02-27

(73) [Sensing and Steering Stereotypes: Extracting and Applying Gender Representation Vectors in LLMs](http://arxiv.org/abs/2502.19721v1), 2025-02-27

(74) [Teaching Dense Retrieval Models to Specialize with Listwise Distillation and LLM Data Augmentation](http://arxiv.org/abs/2502.19712v1), 2025-02-27

(75) [AoECR: AI-ization of Elderly Care Robot](http://arxiv.org/abs/2502.19706v1), 2025-02-27

(76) [M-LLM Based Video Frame Selection for Efficient Video Understanding](http://arxiv.org/abs/2502.19680v1), 2025-02-27

(77) [Old Experience Helps: Leveraging Survey Methodology to Improve AI Text Annotation Reliability in Social Sciences](http://arxiv.org/abs/2502.19679v1), 2025-02-27

(78) [Improving Adversarial Transferability in MLLMs via Dynamic Vision-Language Alignment Attack](http://arxiv.org/abs/2502.19672v1), 2025-02-27

(79) [Investigating Neurons and Heads in Transformer-based LLMs for Typographical Errors](http://arxiv.org/abs/2502.19669v1), 2025-02-27

(80) [SuPreME: A Supervised Pre-training Framework for Multimodal ECG Representation Learning](http://arxiv.org/abs/2502.19668v1), 2025-02-27

(81) [HALO: Hardware-aware quantization with low critical-path-delay weights for LLM acceleration](http://arxiv.org/abs/2502.19662v1), 2025-02-27

(82) [Taxonomy, Opportunities, and Challenges of Representation Engineering for Large Language Models](http://arxiv.org/abs/2502.19649v1), 2025-02-27

(83) [Agentic Mixture-of-Workflows for Multi-Modal Chemical Search](http://arxiv.org/abs/2502.19629v1), 2025-02-26

(84) [Treatment Non-Adherence Bias in Clinical Machine Learning: A Real-World Study on Hypertension Medication](http://arxiv.org/abs/2502.19625v1), 2025-02-26

(85) [Weaker LLMs' Opinions Also Matter: Mixture of Opinions Enhances LLM's Mathematical Reasoning](http://arxiv.org/abs/2502.19622v1), 2025-02-26

(86) [Is Your Paper Being Reviewed by an LLM? A New Benchmark Dataset and Approach for Detecting AI Text in Peer Review](http://arxiv.org/abs/2502.19614v1), 2025-02-26

(87) [Self-rewarding correction for mathematical reasoning](http://arxiv.org/abs/2502.19613v1), 2025-02-26

(88) [Evaluation of Hate Speech Detection Using Large Language Models and Geographical Contextualization](http://arxiv.org/abs/2502.19612v1), 2025-02-26

(89) [Revisiting Word Embeddings in the LLM Era](http://arxiv.org/abs/2502.19607v1), 2025-02-26

(90) [Trustworthy Answers, Messier Data: Bridging the Gap in Low-Resource Retrieval-Augmented Generation for Domain Expert Systems](http://arxiv.org/abs/2502.19596v1), 2025-02-26

(91) [Improving Representation Learning of Complex Critical Care Data with ICU-BERT](http://arxiv.org/abs/2502.19593v1), 2025-02-26

(92) [Where Are We? Evaluating LLM Performance on African Languages](http://arxiv.org/abs/2502.19582v1), 2025-02-26

(93) [Leveraging Retrieval-Augmented Generation and Large Language Models to Predict SERCA-Binding Protein Fragments from Cardiac Proteomics Data](http://arxiv.org/abs/2502.19574v1), 2025-02-26

(94) [Do Large Language Models Know How Much They Know?](http://arxiv.org/abs/2502.19573v1), 2025-02-26

(95) [LORENZA: Enhancing Generalization in Low-Rank Gradient LLM Training via Efficient Zeroth-Order Adaptive SAM](http://arxiv.org/abs/2502.19571v1), 2025-02-26

(96) [Stay Focused: Problem Drift in Multi-Agent Debate](http://arxiv.org/abs/2502.19559v1), 2025-02-26

(97) [Distill Not Only Data but Also Rewards: Can Smaller Language Models Surpass Larger Ones?](http://arxiv.org/abs/2502.19557v1), 2025-02-26

(98) [When Large Language Models Meet Speech: A Survey on Integration Approaches](http://arxiv.org/abs/2502.19548v1), 2025-02-26

(99) [Repurposing the scientific literature with vision-language models](http://arxiv.org/abs/2502.19546v1), 2025-02-26

(100) [Winning Big with Small Models: Knowledge Distillation vs. Self-Training for Reducing Hallucination in QA Agents](http://arxiv.org/abs/2502.19545v1), 2025-02-26
